.github/workflows/ci.yml

```diff
@@ -37,8 +37,19 @@ jobs:
     - name: Test build
       run: go build -o fakemachine cmd/fakemachine/main.go
 
-    - name: Run unit tests
-      run: go test -v ./... | tee test.out
+    - name: Run unit tests (auto backend)
+      run: go test -v ./... | tee -a test.out
+
+    # kvm backend isn't supported in GitHub actions
+    # see: https://github.com/actions/runner-images/issues/183
+    #- name: Run unit tests (kvm backend)
+    #  run: go test -v ./... --backend=kvm | tee -a test.out
+
+    - name: Run unit tests (uml backend)
+      run: go test -v ./... --backend=uml | tee -a test.out
```

Comment:
Potentially this would be better in the matrix which would allow for some parallelism ?